src/librustc/mir/interpret/allocation.rs

```diff
@@ -590,7 +589,7 @@ pub struct AllocationDefinedness {
 
 /// Transferring the definedness mask to other allocations.
 impl<Tag, Extra> Allocation<Tag, Extra> {
-    /// Creates a run-length encoding of the undef_mask.
+    /// Creates a run-length encoding of the undef mask.
```

Comment:
Revert; this refers to the field with an underscore in the name.

Reply:
Ah, my bad.

Reply:
Ah, my bad.

Reply:
Actually, I'm going to stick with "undef mask", since the field is private, and it's written in that form elsewhere.